### PR TITLE
Update job fair with details and link to it on the website

### PR DESCRIPTION
**Pull request by ericholscher**

### CODE REVIEW

File: docs/_data/config-prague-2018.yaml [hunk 1]
```diff
@@ -81,15 +81,15 @@ about:
   mainroom:
     Main Ballroom
   unconfroom:
-    Unconference Room, downstairs
+    Unconference Room
```
Comment (thatdocslady) [inline docs/_data/config-prague-2018.yaml]: ", across the main ballroom"?

File: docs/conf/prague/2018/job-fair.rst [hunk 2]
```diff
@@ -0,0 +1,11 @@
+:template: {{year}}/generic.html
+
+Job Fair
+========
+
+.. include:: /include/conf/events/job-fair.rst
```
Comment (thatdocslady) [inline docs/conf/prague/2018/job-fair.rst:6]: btw there's a typo in the job fair snippet... see if you can spot it ;) (hint: it's a food joke)

File: docs/conf/prague/2018/schedule.rst [hunk 3]
```diff
@@ -140,13 +139,15 @@ Conference Talks
...
+
 Job Fair
 ~~~~~~~~
 
 New in 2018! We'll be holding a job fair on Tuesday morning!
 
 * **Where**: {{about.venue}}, {{about.unconfroom}}
-* **When**: **9:00-12:00**
+* **When**: **10:30-12:10**
 * **Details**: TBA
```
Comment (thatdocslady) [inline docs/conf/prague/2018/schedule.rst]: should we add a link here to the job fair page?

File: docs/conf/prague/2018/sponsors/prospectus.rst [hunk 4]
```diff
@@ -89,7 +89,7 @@ and open source organizations.
 - Small logo & link on the Write the Docs website
 - Name included in welcome announcement in email newsletters and social media
 - Display 1 promotional (“Swag”) item on the conference swag table (provided by sponsor)
-- **NOTE**: This sponsorship does not include a job fair table. You can purchase one for an additional {{sponsorship.job_fair.half}} (50% off)
+- **NOTE**: A table at the job fair is now included in this sponsorship
```
Comment (thatdocslady) [inline docs/conf/prague/2018/sponsors/prospectus.rst:92]: as a special offering for our first year? (maybe?)

File: docs/conf/prague/2018/sponsors/prospectus.rst [hunk 5]
```diff
@@ -182,11 +182,11 @@ Our job fair is a wonderful place to connect with the talented Write the Docs at
 We'll have over {{about.attendees}} people in attendance for our conference,
 and many of those people will come to our job fair to look for new roles and positions.
 
-**{{sponsorship.job_fair.price}}** (a la carte price, also included or discounted with other sponsorship packages)
+It is included in all sponsorship packages this year.
```
Comment (thatdocslady) [inline docs/conf/prague/2018/sponsors/prospectus.rst:185]: even in first draft? I thought that it's just from second draft and upwards...

File: docs/conf/prague/2018/sponsors/prospectus.rst [hunk 6]
```diff
@@ -182,11 +182,11 @@ Our job fair is a wonderful place to connect with the talented Write the Docs at
...
 
-- **Logistics**: The Job Fair will be Tuesday morning in {{job_fair.location}}. It will run from 9am-11:30pm. You can set up from 8am.
+- **Logistics**: The Job Fair will be Tuesday morning in {{job_fair.location}}. You can see the timing details on the :ref:`schedule <{{shortcode}}-{{year}}-job-fair>`.
 
-- **Layout**: Every participant will have a six-foot table and two chairs reserved for them. We do not use booths during the Job Fair. You are welcome to have a standing banner or customized tablecloth. Giveaways/flyers are welcome, please make sure everthing fits on your alloted table.
+- **Layout**: Every participant will have a table and two chairs reserved for them. We do not use booths during the Job Fair. Giveaways/flyers are welcome, please make sure everthing fits on your alloted table.
```
Comment (thatdocslady) [inline docs/conf/prague/2018/sponsors/prospectus.rst]: , > ; ?

**Review by thatdocslady — COMMENTED** [review]
Looks good! Bunch-o-nits, take what you see fit ;)

File: docs/conf/prague/2018/sponsors/prospectus.rst (hunk 5 above)
Comment (ericholscher) [reply docs/conf/prague/2018/sponsors/prospectus.rst:185]: I think the more the merrier -- I doubt the first draft folks will use it, but no real reason not to.

**Comment by plaindocs** [comment]
:100: !

**Pull request merged**